### PR TITLE
Update install cli script docs

### PR DESCRIPTION
**Pull request by schoren**
This PR updates the CLI install script to support apt and yum repos. It also adds docs on how to use those repos manually

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

File: .goreleaser.yaml [hunk 1]
```diff
@@ -33,14 +40,24 @@ nfpms:
 - id: tracetest
   vendor: Kubeshop
   homepage: https://tracetest.kubeshop.io/
+  maintainer: Sebastian Choren <sebastian@kubeshop.io>
```
Comment (xoscar) [inline .goreleaser.yaml:43]: Should we add @mathnogueira as well? 😄

File: docs/installing.md [hunk 2]
```diff
@@ -94,19 +94,55 @@ helm delete tracetest
...
+```sh
+# requirements for our deb repo to work
+sudo apt-get update && sudo apt-get install -y apt-transport-https ca-certificates
+
+# add repo
+echo "deb [trusted=yes] https://apt.fury.io/tracetest/ /" | sudo tee /etc/apt/sources.list.d/fury.list
+
+# update and install
+sudo apt-get update
+sudo apt-get install tracetest
```
Comment (xoscar) [inline docs/installing.md:128]: 🔥

**Review by mathnogueira — APPROVED** [review]
Great work!

**Review by xoscar — APPROVED** [review]
LGTM!

**Pull request merged**